<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog</class>
 <widget class="QDialog" name="dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>269</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Priv</string>
  </property>
  <widget class="QLabel" name="LB_roleName">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>role name</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CMB_roleId">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="LB_formName">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>form name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Form</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CMB_formId">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Action </string>
   </property>
  </widget>
  <widget class="QLabel" name="User">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Role</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CMB_actionId">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_createPrivilage">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>210</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Privilage</string>
   </property>
  </widget>
  <widget class="QLabel" name="LB_actionName">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>140</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>action</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
